src/routes/paths/ApiDelegationPath.tsx

```diff
@@ -7,5 +7,4 @@ export enum RouterPath {
   Receipt = 'receipt',
   ReceivedApiDelegations = 'received-api-delegations',
   Profile = 'ui/Profile',
```

Comment:
Should Profile perhaps be in GeneralPaths?

Reply:
Yes!